### ISSUE TITLE
psycopg2 became dependency

### ISSUE THREAD

**Issue by mtmail** [openstreetmap/Nominatim#1661]
I followed the http://nominatim.org/release-docs/develop/appendix/Install-on-Ubuntu-18/ instructions, but skipped the optional block

```
If you want to run the test suite, you need to install the following additional packages:

sudo apt-get install -y python3-setuptools python3-dev python3-pip \
                        python3-psycopg2 python3-tidylib phpunit php-cgi
```

During import it fails

```
2020-02-04 17:00:09 == Index ranks 0 - 4
/home/vagrant/Nominatim/nominatim/nominatim.py:268: SyntaxWarning: assertion is always true, perhaps remove parentheses?
  assert(False, "Unreachable code")
Traceback (most recent call last):
  File "/home/vagrant/Nominatim/nominatim/nominatim.py", line 31, in <module>
    import psycopg2
ModuleNotFoundError: No module named 'psycopg2'
```

**Comment by lonvia** [comment]
I changed the README but forgot the Vagrant scripts. Every distribution should have packages for that.

**Comment by lonvia** [comment]
Fixed in #1666.
